src/textual/widgets/_loading_indicator.py

```diff
@@ -56,6 +58,12 @@ def _on_mount(self, _: Mount) -> None:
         self._start_time = time()
         self.auto_refresh = 1 / 16
 
+    @on(InputEvent)
+    def on_input(self, event: InputEvent) -> None:
+        """Prevent all input events from bubbling, thus disabling widgets in a loading state."""
+        event.stop()
```

Comment:
@darrenburns Should we do app.bell here? Seems just the error condition one might use it for.

Reply:
I'm not Darren, but does Textual use the terminal bell in any other contexts? Personally I don't think trying to interact with a loading widget should make a fuss about it.

Reply:
We play the bell when clicking on disabled footer keys, and when input is restricted. So there is precedence.

I suspect bell makes more sense in disabled widgets.